specification/apicenter/DataApi/resources/apiDefinitions.tsp

```diff
@@ -0,0 +1,35 @@
+import "@typespec/rest";
+import "../models/apiDefinition.tsp";
+import "../models/exportSpecification.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+using ApiCenter.DataApi.Models;
+
+@versioned(ApiCenter.DataApi.Versions)
+namespace ApiCenter.DataApi;
+
+interface ApiDefinitions {
+  @doc("List a collection of API definitions.")
+  listDefinitions is Operations.ResourceList<
+    ApiDefinition,
+    ListQueryParametersTrait<StandardListQueryParameters>
```

Comment:
Are we sure the support on server-side pagenation via `skip`, `top` (instead of e.g. `continuationToken`)?